src/App.tsx

```diff
@@ -26,70 +27,136 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  switch (sortType) {
+    case 1:
+      visibleGoods.sort((g1, g2) => {
+        return g1.localeCompare(g2);
+      });
+
+      break;
+
+    case 2:
+      visibleGoods.sort((g1, g2) => {
+        return g1.length - g2.length;
+      });
+
+      break;
+
+    default:
+      break;
+  }
+
+  if (isReversed) {
+    return [...visibleGoods].reverse();
+  }
 
   return visibleGoods;
 }
 
-// DON'T save goods to the state
-// type State = {
-//   isReversed: boolean,
-//   sortType: SortType,
-// };
-
-export const App: React.FC = () => {
-  return (
-    <div className="section content">
-      <div className="buttons">
-        <button
-          type="button"
-          className="button is-info is-light"
-        >
-          Sort alphabetically
-        </button>
-
-        <button
-          type="button"
-          className="button is-success is-light"
-        >
-          Sort by length
-        </button>
-
-        <button
-          type="button"
-          className="button is-warning is-light"
-        >
-          Reverse
-        </button>
-
-        <button
-          type="button"
-          className="button is-danger is-light"
-        >
-          Reset
-        </button>
-      </div>
+export class App extends React.Component {
+  state: ReorderOptions = {
+    sortType: 0,
```

Comment:
```suggestion
    sortType: SortType.NONE
```

use enum `SortType` for `sortType`